src/GregClient/GregClient.csproj

```diff
@@ -41,15 +41,8 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' OR '$(TargetFramework)' == 'net48'">
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.IO.Compression" />
-    <Reference Include="System.IO.Compression.FileSystem" />
-    <Reference Include="System.Web" />
-  </ItemGroup>
   <ItemGroup Label="Common package dependencies">
-    <PackageReference Include="RestSharp" Version="106.12.0" />
-	<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="RestSharp" Version="110.2.0" />
```

Comment:
Do we need to update this in Dynamo too ?

Reply:
yes, almost certainly, see my TODO above.